r/fratPhyloseq.Rmd

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Project directory/file naming convention is to use lowercase and hyphens for spaces... so like `frat-phyloseq.Rmd` or just `phyloseq.Rmd`.